tests/HelpersTest.phpt

```diff
@@ -5,6 +5,7 @@ use Simplex\Helpers;
 
 require_once __DIR__ . '/bootstrap.php';
 
+set_time_limit(5);
```

Comment:
Please remove this line as it's a unnecessary side-effect in tests.

Reply:
Time limit was added to confirm that the new test wouldn't get through without the improved code (TDD in micro scale ;) but it's totally unnecessary with the new code.